scripts/qa_crowbarsetup.sh

```diff
@@ -2380,6 +2380,7 @@ function custom_configuration
             if [[ $want_osprofiler = 1 ]] ; then
                 if iscloudver 9M11plus ; then
                     proposal_set_value keystone default "['attributes']['keystone']['osprofiler']['enabled']" "true"
+                    proposal_set_value keystone default "['attributes']['keystone']['osprofiler']['hmac_keys']" "[\"SECRET\"]"
```

Comment:
I think you could just use single quotes instead of escaping the double quotes

Reply:
either `'["SECRET"]'` or `"['SECRET']"` should work.

Reply:
Done